lib/convertKilometerToMach.test.js

```diff
@@ -0,0 +1,15 @@
+const { convertKilometerToMach } = require("../lib");
```

Comment:
I think this file is named improperly. shouldn't have the `test` in it. It should also not be the tests ;)